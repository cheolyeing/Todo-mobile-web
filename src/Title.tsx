import React from "react";

function Title() {
  const TITLE = "THIS WEEK";
  const SUBTITLE = "신나는 일주일을 계획합시다!";
  return (
    <div>
      <h1>{TITLE}</h1>
      <p>{SUBTITLE}</p>
    </div>
  );
}

export default Title;
