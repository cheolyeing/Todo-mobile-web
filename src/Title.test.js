import React from "react";
import { render } from "@testing-library/react";

import Title from "./Title";

describe("<Title />", () => {
  it("matches snapshot", () => {
    const utils = render(<Title />);
    expect(utils.container).toMatchSnapshot();
  });
  it("shows the props correctly", () => {
    const utils = render(<Title />);
    utils.getByText("THIS WEEK");
    utils.getByText("신나는 일주일을 계획합시다!");
  });
});
