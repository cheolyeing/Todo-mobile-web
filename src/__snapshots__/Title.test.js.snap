// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> matches snapshot 1`] = `
<div>
  <div>
    <h1>
      THIS WEEK
    </h1>
    <p>
      신나는 일주일을 계획합시다!
    </p>
  </div>
</div>
`;
